README.md

```diff
@@ -50,6 +46,12 @@ $ brew install dep
 $ brew upgrade dep
 ```
 
+You can also install dep via
```

Comment:
let's further advise against this by making the sentence read:

"If you're interested in hacking on `dep`, you can install via `go get`:"